frontend/workflows/projectSelector/src/dash-hooks.tsx

```diff
@@ -1,9 +1,13 @@
 import * as React from "react";
+import type { clutch as IClutch } from "@clutch-sh/api";
 
-import type { DashAction, DashState } from "./types";
+import type { DashAction, DashState, TimelineAction, TimelineState } from "./types";
 
 export const DashStateContext = React.createContext<DashState | undefined>(undefined);
-
+export const TimelineStateContext = React.createContext<TimelineState | undefined>(undefined);
+export const TimelineUpdateContext = React.createContext<
+  ((action: TimelineAction) => void) | undefined
+>(undefined);
```

Comment:
nit can these be moved under `DashDispatchContext` so that related items stay together?